src/search/components/SearchResultItem.tsx

```diff
@@ -64,14 +65,42 @@ const SearchResultItem: FunctionComponent<SearchResultItemProps> = ({
 			/>
 		);
 	};
+	const checkIfResultIsNotAnItem = (result: Avo.Search.ResultItem): boolean => {
+		return (
+			result.administrative_type === 'collectie' ||
+			result.administrative_type === 'bundel' ||
+			result.administrative_type === 'opdracht'
```

Comment:
Can this be written shorter so you don't have to write `result.administrative_type` 3x?
Something like `result.administrative_type === ('collectie' || 'bundel' || 'opdracht')` ?